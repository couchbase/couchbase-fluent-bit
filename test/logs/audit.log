{"bucket":"default","description":"The specified bucket was selected","id":20492,"name":"select bucket","peername":"127.0.0.1:56021","real_userid":{"domain":"local","user":"@ns_server"},"sockname":"127.0.0.1:11209","timestamp":"2021-03-09T20:12:17.445039Z"}
{"description":"An HTTP request was made to the API at /admin/stats.","httpMethod":"GET","httpResultCode":200,"id":28689,"name":"/admin/stats API request","real_userid":{"domain":"builtin","user":"@"},"remote":{"ip":"127.0.0.1","port":53465},"timestamp":"2021-03-09T20:12:17.489Z"}
{"context":"<nil>","description":"Eventing function draft definitions were read","id":32772,"name":"Fetch Drafts","real_userid":{"domain":"builtin","user":"@"},"timestamp":"2021-03-09T20:12:17.713Z"}
