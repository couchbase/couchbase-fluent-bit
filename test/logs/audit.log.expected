/fluent-bit/test/logs/audit.log.actual: [1615320737.445039000, {"file_path":"/fluent-bit/test/logs","file":"audit.log","bucket":"default","description":"The specified bucket was selected","timestamp":"2021-03-09T20:12:17.445039Z","id":20492,"name":"select bucket","pod":{"name":"test-pod","namespace":"default"},"peername":"127.0.0.1:56021","couchbase":{"logging.version":"99","cluster":"test-cluster","node":"cb-node.test-cluster.default.svc"},"real_userid":{"user":"@ns_server","domain":"local"},"fluentbit.version":"99","logshipper":"couchbase.sidecar.fluentbit","hostname":"test-container","sockname":"127.0.0.1:11209","filename":"/fluent-bit/test/logs/audit.log","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/audit.log.actual: [1615320737.489000000, {"file_path":"/fluent-bit/test/logs","file":"audit.log","hostname":"test-container","description":"An HTTP request was made to the API at /admin/stats.","timestamp":"2021-03-09T20:12:17.489Z","id":28689,"name":"/admin/stats API request","pod":{"name":"test-pod","namespace":"default"},"couchbase":{"logging.version":"99","cluster":"test-cluster","node":"cb-node.test-cluster.default.svc"},"fluentbit.version":"99","real_userid":{"user":"@","domain":"builtin"},"logshipper":"couchbase.sidecar.fluentbit","remote":{"ip":"127.0.0.1","port":53465},"httpMethod":"GET","httpResultCode":200,"filename":"/fluent-bit/test/logs/audit.log","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/audit.log.actual: [1615320737.713000000, {"file_path":"/fluent-bit/test/logs","file":"audit.log","hostname":"test-container","description":"Eventing function draft definitions were read","timestamp":"2021-03-09T20:12:17.713Z","id":32772,"name":"Fetch Drafts","pod":{"name":"test-pod","namespace":"default"},"context":"<nil>","real_userid":{"user":"@","domain":"builtin"},"couchbase":{"logging.version":"99","cluster":"test-cluster","node":"cb-node.test-cluster.default.svc"},"fluentbit.version":"99","logshipper":"couchbase.sidecar.fluentbit","filename":"/fluent-bit/test/logs/audit.log","level":"INFO","level_numeric":"20"}]
